pkg/cache/v3/status.go

```diff
@@ -101,8 +100,8 @@ type DeltaResponseWatch struct {
 	// Response is the channel to push the delta responses to
 	Response chan DeltaResponse
 
-	// VersionMap for the stream
-	StreamState stream.StreamState
+	// State for the type subscription
```

Comment:
Godoc format please, e.g.:

```suggestion
	// Subscription stores the current client subscriptions.
```

Reply:
Updated